src/contracts/Content/Form/Provider/GroupedContentFormFieldsProviderInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\ContentForms\Content\Form\Provider;
+
+interface GroupedContentFormFieldsProviderInterface
+{
+    /**
+     * @param \Symfony\Component\Form\FormInterface[] $fieldsDataForm
+     * @psalm-return array<string, array<int, string>> Array of fieldGroupIdentifier grouped by fieldGroupName.
```

Comment:
Are we using psalm anywhere?

```suggestion
     * @phpstan-return array<string, array<int, string>> Array of fieldGroupIdentifier grouped by fieldGroupName.
```